mmeds/spawn.py

```diff
@@ -374,7 +389,7 @@ def run(self):
             self.count += 1
             # If there is nothing in the process queue, sleep
             if self.q.empty():
-                if self.count == 20:
+                if self.count >= 20:
```

Comment:
20, or 42, or something else. Why 20, what does this magic number represent?

Reply:
Completely arbitrary, as you say. David put this in so that the watcher would sleep if nothing was coming down the pipe for that many steps of the watcher loop. I switched it from `==` to `>=` because I was worried about some edge case where the `self.count` would pass 20 before being able to check it.

Reply:
Whenever you have this type of situation, parametrize the function and set a default value or set a constant in the function itself. That gives a bit more of info of what the value represents.